### PR TITLE
Resolve the cwd and return the result from the parse method

### PR DESCRIPTION
**Pull request by mupkoo**
There were two things that were not ok.

* The `cwd` was set to the file name
* There was no return infront of `helpers.parse`

This PR should fix those problems

### CODE REVIEW

**Comment by mupkoo** [comment]
This should fix the #23

**Comment by k2b6s9j** [comment]
I'll have to test later, but iirc there is no need for a return in front of `helpers.parse`. It might actually cause issue iirc.

**Comment by mupkoo** [comment]
The linter messages are not shown if there is no return statement.

File: lib/main.js [hunk 1]
```diff
@@ -38,10 +39,11 @@ export default {
...
       lint: (activeEditor) => {
         const command = atom.config.get("linter-ruby.rubyExecutablePath");
-        const file = activeEditor.getPath()
+        const file = activeEditor.getPath();
+        const cwd = path.dirname(file);
         const args = ["-wc", file];
-        return helpers.exec(command, args, {stream: "stderr", cwd: file}).then(output =>
-          helpers.parse(output, regex)
+        return helpers.exec(command, args, {stream: "stderr", cwd: cwd}).then(output =>
+          return helpers.parse(output, regex)
```
Comment (steelbrain) [inline lib/main.js]: return on this line is unnecessary
```js
a.filter(x => x * x)
```
is equal to
```js
a.filter(function(x){ return x * x })
```

**Comment by mupkoo** [comment]
I removed the return statement. On my local machine I was using a different syntax.

```javascript
...then((output) => {
   return ....
});
```

**Comment by k2b6s9j** [comment]
This will be published in the next path update. Thanks.

File: lib/main.js (hunk 1 above)
Comment (mupkoo) [inline lib/main.js]: Thanks for the explanation @steelbrain :). You learn something new everyday :)

**Pull request merged**